docker/alpine/docker-start

```diff
@@ -5,7 +5,7 @@ set -e
 ##
 # For volume mounts...
 ##
-chown -R frr:frr /etc/frr
+chown -R frr:frr /etc/frr || true
```

Comment:
if I understand correctly that happens only on read-only mounted filesystems or what is the actuall error returned?